WebApplication1/Application/Commands/UserUpdateCommand.cs

```diff
@@ -0,0 +1,31 @@
+﻿using AutoMapper;
+using MediatR;
+using System.Threading;
+using System.Threading.Tasks;
+using WebApplication1.Application.Options;
+using WebApplication1.Data;
+using WebApplication1.Domain;
+using WebApplication1.DTO;
+
+namespace WebApplication1.Application.Commands
+{
+    public class UserUpdateCommand : IRequest<bool>
+    {
+        public long Id { get; set; }
+        public UserDto UserDto { get; set; }
```

Comment:
do not use UserDto here, use just properties
```suggestion
        public long Id { get; set; }
        public string FirstName { get; set; }
        .....
```

Reply:
the same for AddCommand